[0m13:59:42.144171 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104100c10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104124f10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1041256d0>]}


============================== 13:59:42.146890 | 50b0a223-1ffb-40c8-9630-af11759389cf ==============================
[0m13:59:42.146890 [info ] [MainThread]: Running with dbt=1.8.7
[0m13:59:42.147184 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init users', 'send_anonymous_usage_stats': 'True'}
[0m13:59:42.154944 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/Users/yoonmoon/.dbt')}
[0m13:59:42.155231 [info ] [MainThread]: Creating dbt configuration folder at 
[0m13:59:42.156300 [debug] [MainThread]: Starter project path: /Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/include/starter_project
[0m13:59:42.160405 [info ] [MainThread]: 
Your new dbt project "users" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m13:59:42.160607 [info ] [MainThread]: Setting up your profile.
[0m14:01:49.964954 [info ] [MainThread]: Profile users written to /Users/yoonmoon/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m14:01:49.995902 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 127.88655, "process_user_time": 0.555495, "process_kernel_time": 0.137441, "process_mem_max_rss": "103219200", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:01:49.996879 [debug] [MainThread]: Command `dbt init` succeeded at 14:01:49.996755 after 127.89 seconds
[0m14:01:49.997322 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040bc150>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040e3ad0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1008b9890>]}
[0m14:01:49.997771 [debug] [MainThread]: Flushing usage events
[0m14:02:19.439828 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x124b78c10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x123360790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x124b9dbd0>]}


============================== 14:02:19.442765 | ee44c6fc-4fbb-400a-ad2e-43ab7e64040e ==============================
[0m14:02:19.442765 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:02:19.443100 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:02:19.451630 [info ] [MainThread]: dbt version: 1.8.7
[0m14:02:19.451863 [info ] [MainThread]: python version: 3.11.7
[0m14:02:19.452019 [info ] [MainThread]: python path: /Users/yoonmoon/git/learn-dbt/dbt-env/bin/python
[0m14:02:19.452153 [info ] [MainThread]: os info: macOS-15.0.1-arm64-arm-64bit
[0m14:02:19.488417 [info ] [MainThread]: Using profiles dir at /Users/yoonmoon/.dbt
[0m14:02:19.488669 [info ] [MainThread]: Using profiles.yml file at /Users/yoonmoon/.dbt/profiles.yml
[0m14:02:19.488811 [info ] [MainThread]: Using dbt_project.yml file at /Users/yoonmoon/git/learn-dbt/dbt_project.yml
[0m14:02:19.489182 [info ] [MainThread]: adapter type: postgres
[0m14:02:19.489313 [info ] [MainThread]: adapter version: 1.8.2
[0m14:02:19.489440 [info ] [MainThread]: Configuration:
[0m14:02:19.489554 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m14:02:19.489660 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m14:02:19.489769 [info ] [MainThread]: Required dependencies:
[0m14:02:19.489909 [debug] [MainThread]: Executing "git --help"
[0m14:02:19.502122 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m14:02:19.502601 [debug] [MainThread]: STDERR: "b''"
[0m14:02:19.502763 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m14:02:19.502911 [info ] [MainThread]: Connection:
[0m14:02:19.503073 [info ] [MainThread]:   host: localhost
[0m14:02:19.503194 [info ] [MainThread]:   port: 5432
[0m14:02:19.503304 [info ] [MainThread]:   user: postgres
[0m14:02:19.503410 [info ] [MainThread]:   database: postgres
[0m14:02:19.503514 [info ] [MainThread]:   schema: source
[0m14:02:19.503621 [info ] [MainThread]:   connect_timeout: 10
[0m14:02:19.503726 [info ] [MainThread]:   role: None
[0m14:02:19.503832 [info ] [MainThread]:   search_path: None
[0m14:02:19.503937 [info ] [MainThread]:   keepalives_idle: 0
[0m14:02:19.504039 [info ] [MainThread]:   sslmode: None
[0m14:02:19.504144 [info ] [MainThread]:   sslcert: None
[0m14:02:19.504244 [info ] [MainThread]:   sslkey: None
[0m14:02:19.504353 [info ] [MainThread]:   sslrootcert: None
[0m14:02:19.504457 [info ] [MainThread]:   application_name: dbt
[0m14:02:19.504559 [info ] [MainThread]:   retries: 1
[0m14:02:19.504846 [info ] [MainThread]: Registered adapter: postgres=1.8.2
[0m14:02:19.528802 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m14:02:19.555572 [debug] [MainThread]: Using postgres connection "debug"
[0m14:02:19.555812 [debug] [MainThread]: On debug: select 1 as id
[0m14:02:19.555953 [debug] [MainThread]: Opening a new connection, currently in state init
[0m14:02:19.666580 [debug] [MainThread]: Postgres adapter: Got a retryable error when attempting to open a postgres connection.
1 attempts remaining. Retrying in 0 seconds.
Error:
connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "postgres"

[0m14:02:19.674604 [debug] [MainThread]: Postgres adapter: Error running SQL: select 1 as id
[0m14:02:19.674905 [debug] [MainThread]: Postgres adapter: Rolling back transaction.
[0m14:02:19.675180 [debug] [MainThread]: On debug: No close available on handle
[0m14:02:19.675422 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m14:02:19.675668 [info ] [MainThread]: [31m2 checks failed:[0m
[0m14:02:19.675930 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/yoonmoon/git/learn-dbt/dbt_project.yml> not found

[0m14:02:19.676092 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  connection to server at "localhost" (::1), port 5432 failed: FATAL:  password authentication failed for user "postgres"
  

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m14:02:19.678221 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.27202126, "process_user_time": 0.564475, "process_kernel_time": 0.149552, "process_mem_max_rss": "113491968", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:02:19.678649 [debug] [MainThread]: Command `dbt debug` failed at 14:02:19.678582 after 0.27 seconds
[0m14:02:19.678884 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m14:02:19.679079 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x124b84490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x124b34290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x122b1df90>]}
[0m14:02:19.679310 [debug] [MainThread]: Flushing usage events
[0m14:03:00.158051 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104511b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10452d790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10452df50>]}


============================== 14:03:00.160698 | 95f964f4-759c-43af-b453-7b8d01276b3c ==============================
[0m14:03:00.160698 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:03:00.161002 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m14:03:00.161312 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/yoonmoon/git/learn-dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m14:03:00.162718 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.03457575, "process_user_time": 0.516119, "process_kernel_time": 0.097243, "process_mem_max_rss": "99876864", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:03:00.163063 [debug] [MainThread]: Command `dbt run` failed at 14:03:00.163009 after 0.03 seconds
[0m14:03:00.163263 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104143350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1044c40d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x100cc1890>]}
[0m14:03:00.163446 [debug] [MainThread]: Flushing usage events
[0m14:14:13.700470 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104491d50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104501050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104501d50>]}


============================== 14:14:13.703238 | ba379d24-4218-49a4-b904-e065d5a9af9e ==============================
[0m14:14:13.703238 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:14:13.703565 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init dbt', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:14:20.768479 [error] [MainThread]: Encountered an error:

[0m14:14:20.774629 [error] [MainThread]: Traceback (most recent call last):
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/cli/main.py", line 470, in init
    results = task.run()
              ^^^^^^^^^^
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/task/init.py", line 325, in run
    project_name = self.get_valid_project_name()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/task/init.py", line 279, in get_valid_project_name
    name = click.prompt("Enter a name for your project (letters, digits, underscore)")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m14:14:20.777574 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 7.1100545, "process_user_time": 0.521997, "process_kernel_time": 0.112544, "process_mem_max_rss": "100548608", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:14:20.778206 [debug] [MainThread]: Command `dbt init` failed at 14:14:20.778107 after 7.11 seconds
[0m14:14:20.778633 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1045017d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104498090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x100dd8f50>]}
[0m14:14:20.778894 [debug] [MainThread]: Flushing usage events
[0m14:14:35.647236 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10be702d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10be67d10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10be43f10>]}


============================== 14:14:35.650062 | 14051312-f98e-4b40-a7c9-9732b9e39fa8 ==============================
[0m14:14:35.650062 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:14:35.650369 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt init test1', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:14:35.658878 [debug] [MainThread]: Starter project path: /Users/yoonmoon/git/learn-dbt/dbt-env/lib/python3.11/site-packages/dbt/include/starter_project
[0m14:14:35.662923 [info ] [MainThread]: 
Your new dbt project "test1" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:14:35.663154 [info ] [MainThread]: Setting up your profile.
[0m14:15:07.103364 [info ] [MainThread]: Profile test1 written to /Users/yoonmoon/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m14:15:07.107306 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 31.49239, "process_user_time": 0.562425, "process_kernel_time": 0.124121, "process_mem_max_rss": "104300544", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:15:07.108099 [debug] [MainThread]: Command `dbt init` succeeded at 14:15:07.107987 after 31.49 seconds
[0m14:15:07.108526 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10be20290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1056cd910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105964ed0>]}
[0m14:15:07.108944 [debug] [MainThread]: Flushing usage events
[0m14:15:15.418132 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105a56e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105ac4fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105ac5790>]}


============================== 14:15:15.420637 | f97e7081-9131-4f40-a12a-ead7d99adf83 ==============================
[0m14:15:15.420637 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:15:15.420918 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:15:15.428122 [info ] [MainThread]: dbt version: 1.8.7
[0m14:15:15.428328 [info ] [MainThread]: python version: 3.11.7
[0m14:15:15.428477 [info ] [MainThread]: python path: /Users/yoonmoon/git/learn-dbt/dbt-env/bin/python
[0m14:15:15.428608 [info ] [MainThread]: os info: macOS-15.0.1-arm64-arm-64bit
[0m14:15:15.455370 [info ] [MainThread]: Using profiles dir at /Users/yoonmoon/.dbt
[0m14:15:15.455629 [info ] [MainThread]: Using profiles.yml file at /Users/yoonmoon/.dbt/profiles.yml
[0m14:15:15.455819 [info ] [MainThread]: Using dbt_project.yml file at /Users/yoonmoon/git/learn-dbt/dbt_project.yml
[0m14:15:15.456001 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m14:15:15.457380 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.06843683, "process_user_time": 0.548158, "process_kernel_time": 0.102218, "process_mem_max_rss": "102367232", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:15:15.457648 [debug] [MainThread]: Command `dbt debug` failed at 14:15:15.457605 after 0.07 seconds
[0m14:15:15.457825 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105aac110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105a5c110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10125ce10>]}
[0m14:15:15.457998 [debug] [MainThread]: Flushing usage events
[0m13:22:02.172628 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10452f1d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10459d090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10459d7d0>]}


============================== 13:22:02.175545 | b0e4e95d-4866-472a-82f9-c40b7c4c949b ==============================
[0m13:22:02.175545 [info ] [MainThread]: Running with dbt=1.8.7
[0m13:22:02.175853 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m13:22:02.184505 [info ] [MainThread]: dbt version: 1.8.7
[0m13:22:02.184741 [info ] [MainThread]: python version: 3.11.7
[0m13:22:02.184911 [info ] [MainThread]: python path: /Users/yoonmoon/git/learn-dbt/dbt-env/bin/python
[0m13:22:02.185194 [info ] [MainThread]: os info: macOS-15.0.1-arm64-arm-64bit
[0m13:22:02.224827 [info ] [MainThread]: Using profiles dir at /Users/yoonmoon/.dbt
[0m13:22:02.225102 [info ] [MainThread]: Using profiles.yml file at /Users/yoonmoon/.dbt/profiles.yml
[0m13:22:02.225241 [info ] [MainThread]: Using dbt_project.yml file at /Users/yoonmoon/git/learn-dbt/dbt_project.yml
[0m13:22:02.225419 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m13:22:02.226977 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.087263875, "process_user_time": 0.547738, "process_kernel_time": 0.126159, "process_mem_max_rss": "102924288", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m13:22:02.227248 [debug] [MainThread]: Command `dbt debug` failed at 13:22:02.227202 after 0.09 seconds
[0m13:22:02.227419 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1041b3350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104534150>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10244f7d0>]}
[0m13:22:02.227588 [debug] [MainThread]: Flushing usage events
[0m23:21:58.858563 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108254e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10828d710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10828de90>]}


============================== 23:21:58.861706 | f39de457-7847-43d0-9f11-eb2133861340 ==============================
[0m23:21:58.861706 [info ] [MainThread]: Running with dbt=1.8.7
[0m23:21:58.862013 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt run --models +hist_all', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m23:21:58.862301 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/yoonmoon/git/learn-dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:21:58.868138 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.042170793, "process_user_time": 0.522569, "process_kernel_time": 0.12764, "process_mem_max_rss": "100614144", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m23:21:58.868449 [debug] [MainThread]: Command `dbt run` failed at 23:21:58.868403 after 0.04 seconds
[0m23:21:58.868649 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10821fe90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10828e250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104a21a10>]}
[0m23:21:58.868833 [debug] [MainThread]: Flushing usage events
[0m12:23:31.925997 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d56dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109dc5110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109dc5810>]}


============================== 12:23:31.928608 | 81874965-6dbe-46c7-b31c-c180586760ef ==============================
[0m12:23:31.928608 [info ] [MainThread]: Running with dbt=1.8.7
[0m12:23:31.928914 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m12:23:31.935999 [info ] [MainThread]: dbt version: 1.8.7
[0m12:23:31.936232 [info ] [MainThread]: python version: 3.11.7
[0m12:23:31.936393 [info ] [MainThread]: python path: /Users/yoonmoon/git/learn-dbt/dbt-env/bin/python
[0m12:23:31.936529 [info ] [MainThread]: os info: macOS-15.0.1-arm64-arm-64bit
[0m12:23:31.965138 [info ] [MainThread]: Using profiles dir at /Users/yoonmoon/.dbt
[0m12:23:31.965391 [info ] [MainThread]: Using profiles.yml file at /Users/yoonmoon/.dbt/profiles.yml
[0m12:23:31.965524 [info ] [MainThread]: Using dbt_project.yml file at /Users/yoonmoon/git/learn-dbt/dbt_project.yml
[0m12:23:31.965690 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m12:23:31.966474 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.068179086, "process_user_time": 0.549753, "process_kernel_time": 0.100199, "process_mem_max_rss": "103710720", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:23:31.966734 [debug] [MainThread]: Command `dbt debug` failed at 12:23:31.966692 after 0.07 seconds
[0m12:23:31.966913 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109d5c050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109dc5b10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1056dd990>]}
[0m12:23:31.967081 [debug] [MainThread]: Flushing usage events
[0m12:23:42.378436 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae1b050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae88fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a8604d0>]}


============================== 12:23:42.380530 | 43ea8541-b913-47cf-900b-645fdddc12db ==============================
[0m12:23:42.380530 [info ] [MainThread]: Running with dbt=1.8.7
[0m12:23:42.380834 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:23:42.384764 [info ] [MainThread]: dbt version: 1.8.7
[0m12:23:42.384952 [info ] [MainThread]: python version: 3.11.7
[0m12:23:42.385157 [info ] [MainThread]: python path: /Users/yoonmoon/git/learn-dbt/dbt-env/bin/python
[0m12:23:42.385374 [info ] [MainThread]: os info: macOS-15.0.1-arm64-arm-64bit
[0m12:23:42.410578 [info ] [MainThread]: Using profiles dir at /Users/yoonmoon/.dbt
[0m12:23:42.410878 [info ] [MainThread]: Using profiles.yml file at /Users/yoonmoon/.dbt/profiles.yml
[0m12:23:42.411019 [info ] [MainThread]: Using dbt_project.yml file at /Users/yoonmoon/git/learn-dbt/dbt_project.yml
[0m12:23:42.411202 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m12:23:42.412128 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.0606125, "process_user_time": 0.544064, "process_kernel_time": 0.084702, "process_mem_max_rss": "103825408", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:23:42.412392 [debug] [MainThread]: Command `dbt debug` failed at 12:23:42.412351 after 0.06 seconds
[0m12:23:42.412573 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae19390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae20290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10537d010>]}
[0m12:23:42.412744 [debug] [MainThread]: Flushing usage events
[0m12:23:47.827501 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108d57190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108dc5990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108dc6090>]}


============================== 12:23:47.829613 | 7892c424-97ce-4acd-9291-6c78c2e7cccf ==============================
[0m12:23:47.829613 [info ] [MainThread]: Running with dbt=1.8.7
[0m12:23:47.829910 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/yoonmoon/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m12:23:47.830186 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/yoonmoon/git/learn-dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:23:47.830963 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.030282374, "process_user_time": 0.516992, "process_kernel_time": 0.06847, "process_mem_max_rss": "101515264", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:23:47.831233 [debug] [MainThread]: Command `dbt run` failed at 12:23:47.831190 after 0.03 seconds
[0m12:23:47.831417 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108d5c1d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108dc5dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105624b90>]}
[0m12:23:47.831595 [debug] [MainThread]: Flushing usage events
